packages/shorebird_code_push_protocol/lib/src/messages/create_release_artifact/create_release_artifact_request.dart

```diff
@@ -38,7 +38,7 @@ class CreateReleaseArtifactRequest {
 
   /// Whether the artifact can installed and run on a device/emulator as-is.
   @JsonKey(fromJson: _parseStringToBool, toJson: _parseBoolToString)
-  final bool canSideload;
+  final bool? canSideload;
```

Comment:
don't we need to re-run build_runner?

Reply:
Just ran and nothing changed – probably because we define our own `toJson` and `fromJson`